storagenode/bandwidth/usage.go

```diff
@@ -77,6 +78,11 @@ func (usage *Usage) Total() int64 {
 		usage.Delete
 }
 
+// CachedBandwidthUsed returns total bandwidth for the current month from cache
+func CachedBandwidthUsed(ctx context.Context, db DB) (int64, error) {
```

Comment:
Not sure whether this extra func is necessary here, because there aren't any defaults that need to be handled.

Reply:
removed